api/clients/v2/verification/cert_verifier_generic.go

```diff
@@ -0,0 +1,236 @@
+package verification
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	clients "github.com/Layr-Labs/eigenda/api/clients/v2"
+	"github.com/Layr-Labs/eigenda/api/clients/v2/coretypes"
+	"github.com/Layr-Labs/eigenda/common"
+	genericVerifierBinding "github.com/Layr-Labs/eigenda/contracts/bindings/EigenDACertVerifier"
+	opsrbinding "github.com/Layr-Labs/eigenda/contracts/bindings/OperatorStateRetriever"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+)
+
+// CertVerifier is responsible for making eth calls against version agnostic CertVerifier contracts to ensure
+// cryptographic and structural integrity of EigenDA certificate types.
+// The V3 cert verifier contract is located at:
+// https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/periphery/cert/v3/EigenDACertVerifierV3.sol
+type CertVerifier struct {
```

Comment:
File name is still `cert_verifier_generic`, should be `cert_verifier`

Reply:
[92ab00c](https://github.com/Layr-Labs/eigenda/pull/1532/commits/92ab00cee909375647e0c4177761bd799429dfda)